### PR TITLE
made changes to the Fibonacci function

### PR DESCRIPTION
**Pull request by cliff002**
I made adjustments to the Fibonacci function simplifying the code. Added assert test and docstring after reviewing the series_template.py file.

### CODE REVIEW

**Comment by cliff002** [comment]
Fibonacci series exercise completed with 3 functions,  well-formed docstrings, and a series of assert statements.

File: students/clifford_butler/lesson02/series.py [hunk 1]
```diff
@@ -1,15 +1,99 @@
...
+        return 2
+    
+    elif (n == 1):
+        return 1
+    
+    else:
+        return lucas(n-1) + lucas(n-2)
+    
+ 
+def sum_series(n, a = 0, b = 1):
```
Comment (charlesrobison) [inline students/clifford_butler/lesson02/series.py]: No white space around these assignment operators

File: students/clifford_butler/lesson02/series.py [hunk 2]
```diff
@@ -1,15 +1,99 @@
...
 
+    # test if sum_series works for arbitrary initial values
+    assert sum_series(0, 3, 2) == 3
+    assert sum_series(1, 3, 2) == 2
+    assert sum_series(2, 3, 2) == 5
+    assert sum_series(3, 3, 2) == 7
+    assert sum_series(4, 3, 2) == 12
+    assert sum_series(5, 3, 2) == 19
 
+    print("tests passed")
```
Comment (charlesrobison) [inline students/clifford_butler/lesson02/series.py]: Looks good!

File: students/clifford_butler/lesson03/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    Add another fruit to the beginning of the list using insert() and display the list.
+    Display all the fruits that begin with “P”, using a for loop.
+"""
+
+print ("Series 1 exercise:" + "\n")
+
+# Create a list and display the list
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+for item in (fruit_list):
+    print ("A list has been created \
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: No white space after print

File: students/clifford_butler/lesson03/list_lab.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    fruit_list.append(response)
+else: print("Already in list!")
+print(response,"has been added to the end of the list. Here is the new list:")
+print(str(fruit_list)[1:-1])
+
+
+# Ask the user for a number and display the number back to the user 
+# and the fruit corresponding to that number (on a 1-is-first basis)
+response2 = int(input("What fruit would you like to display? \
+Enter the number corresonding with the fruit: "))
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: len() is a nice addition here

File: students/clifford_butler/lesson03/list_lab.py [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+with the following", len(fruit_list), "fruit:", str(fruit_list)[1:-1])
+    break
+
+
+# Ask the user for another fruit and add it to the end of the list
+response = input("Enter the fruit you would like to add to the list: ")
+if response not in fruit_list:
+    fruit_list.append(response)
+else: print("Already in list!")
+print(response,"has been added to the end of the list. Here is the new list:")
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: White space after comma

File: students/clifford_butler/lesson03/list_lab.py [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+Using the list created in series 1 above:
+
+    Display the list.
+    Remove the last fruit from the list.
+    Display the list.
+    Ask the user for a fruit to delete, find it and delete it.
+    (Bonus: Multiply the list times two. Keep asking until a match is found. Once found, delete all occurrences.)
+"""
+
+print ("\n" + "Series 2 exercise:" + "\n")
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: White space after print

File: students/clifford_butler/lesson03/list_lab.py [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+
+# Display items in fruit_list
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+print ("The list contains the following", len(fruit_list), "fruit:", str(fruit_list)[1:-1],"\n")
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: Here too

File: students/clifford_butler/lesson03/list_lab.py [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+
+# Make a new list with the contents of the original, but with all the letters in each item reversed.
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+series_four_list = fruit_list[:]
+series_four_list2 = [x[::-1] for x in series_four_list][:]
+
+#Delete the last item of the original list. Display the original list and the copy.
+fruit_list = fruit_list[:-1]
+print("The original list has the following", len(fruit_list), "fruit:", str(fruit_list)[1:-1])  
+print("The copied list has the following", len(series_four_list2), "fruit:", str(series_four_list2)[1:-1])
```
Comment (charlesrobison) [inline students/clifford_butler/lesson03/list_lab.py]: Works well!

**Pull request merged**